src/components/Banner/Banner.module.css

```diff
@@ -6,163 +6,186 @@
+  border-top-color: var(--border-dark-color); /* 3 */
 }
 
 /**
- * Banner success
- * 1) The thick left border operates as a foreground band of color,
- *    so foreground rather than border is used
+ * Banner informational icon
+ * 1) Selector doubled up to bump specificity to not interact with .icon.
+ *    See code guidelines for more info.
  */
-.banner--success {
-  background: var(--eds-theme-color-utility-success-background);
-  border-color: var(--eds-theme-color-utility-success-border);
-  border-left-color: var(--eds-theme-color-utility-success-foreground); /* 1 */
-  color: var(--eds-theme-color-utility-success-foreground);
+.banner__icon {
+  color: var(--border-dark-color); /* 3 */
+  --icon-size-default: var(--eds-size-5);
 }
 
-/**
- * Banner warning
- */
-.banner--warning {
-  background: var(--eds-theme-color-utility-warning-background);
-  border-color: var(--eds-theme-color-utility-warning-border);
-  border-left-color: var(--eds-theme-color-utility-warning-foreground); /* 1 */
-  color: var(--eds-theme-color-utility-warning-foreground);
+.banner__textAndAction,
+.banner__textContent {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
 }
 
-/**
- * Banner error
- */
-.banner--error {
-  background: var(--eds-theme-color-utility-error-background);
-  border-color: var(--eds-theme-color-utility-error-border);
-  border-left-color: var(--eds-theme-color-utility-error-foreground); /* 1 */
-  color: var(--eds-theme-color-utility-error-foreground);
+.banner__textAndAction {
+  gap: var(--eds-size-2);
 }
 
-/**
- * Brand banner
- */
-.banner--brand {
-  background: var(--eds-color-brand-blue-100); /* TODO */
-  border-color: var(--eds-theme-color-primary-border);
-  border-left-color: var(--eds-theme-color-primary-foreground); /* 1 */
-  color: var(--eds-theme-color-primary-foreground);
+.banner__textContent {
+  gap: var(--eds-size-1);
+}
+
+.banner__action {
+  justify-self: center;
+  align-self: center;
 }
 
 /**
- * Banner informational icon
- * 1) Selector doubled up to bump specificity to not interact with .icon.
- *    See code guidelines for more info.
+ * Banner close button
+ * 1) Button used to dismiss dismissable banners
+ * 2) Match close button icon color to the banner text color
  */
-.banner__icon.banner__icon {
-  margin-right: var(--eds-size-2);
-  fill: var(--eds-theme-color-neutral-medium-foreground);
-  width: var(--eds-size-3);
-  height: var(--eds-size-3);
+.banner__close-btn.banner__close-btn {
+  position: absolute;
+  top: var(--eds-size-half);
+  right: var(--eds-size-half);
+  /* TODO: handle hover styles */
+  color: inherit; /* 2 */
+}
 
-  /**
-   * Icon within success banner
-   */
-  .banner--success & {
-    fill: var(--eds-theme-color-utility-success-foreground);
-  }
+/*------------------------------------*\
+    # HORIZONTAL ORIENTATION
+\*------------------------------------*/
 
+/* TODO is there a token for this? */
+@media screen and (min-width: 768px) {
```

Comment:
ya you'll need to use the `$` variable syntax because native CSS variables won't work -- but check `breakpoints.js` for reference. I believe this would be `$eds-bp-md`

Reply:
Awesome, thanks!